### ISSUE TITLE
res settings problem

### ISSUE THREAD

**Issue by maniac428** [sorayuki/obs-multi-rtmp#128]
i stream twitch 1664x936 res
and multistream youtube 2560x1440 res

and there are some bugs
twitch works fine but youtube res settings are not work

obs log said youtube is 2560x1440 but in video , seems like 1664x936
so, when i stream youtube only , it works fine

in a nutshell, multi rtmp res settings not working
i use 1080ti nvenc

**Comment by sorayuki** [comment]
Yes, it's not good setting to configure output size larger than render size.
I suggest that set OBS's render size to the maximium among output sizes, or you will not get ideal quality.
